spec/rubocop/cop/lint/to_json_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Lint::ToJSON do
+  subject(:cop) { described_class.new(config) }
+
+  let(:config) { RuboCop::Config.new }
+
+  it 'registers an offense when using `#to_json` without arguments' do
+    expect_offense(<<-RUBY.strip_indent)
+      def to_json
+      ^^^^^^^^^^^ `#to_json` requires an optional hash argument to be parsable via JSON.generate(obj).
+      end
+    RUBY
+  end
+
+  it 'does not register an offense when using `#to_json` with arguments' do
+    expect_no_offenses(<<-RUBY.strip_indent)
+    def to_json(opts = {})
```

Comment:
Let's indent the contents of the heredoc one more level for this and the next example. 🙂